cdap-proto/src/test/java/co/cask/cdap/proto/security/AuthorizableTest.java

```diff
@@ -69,15 +91,47 @@ public void testFromEntity() {
     } catch (UnsupportedOperationException e) {
       // expected
     }
+  }
 
-    programId = appId.program(ProgramType.MAPREDUCE, "prog");
-    authorizable = Authorizable.fromEntityId(programId);
-    // drop the version while asserting
-    Assert.assertEquals(programId.toString().replace(".1.0-SNAPSHOT", ""), authorizable.toString());
-
+  @Test
+  public void testPrincipal() {
     KerberosPrincipalId kerberosPrincipalId = new KerberosPrincipalId("eve/host1.com@domain.net");
-    authorizable = Authorizable.fromEntityId(kerberosPrincipalId);
-    // drop the version while asserting
+    Authorizable authorizable = Authorizable.fromEntityId(kerberosPrincipalId);
     Assert.assertEquals(kerberosPrincipalId.toString(), authorizable.toString());
   }
+
+  @Test
+  public void testStream() {
+    StreamId streamId = new StreamId("ns", "stream");
+    Authorizable authorizable = Authorizable.fromEntityId(streamId);
```

Comment:
>Unless we have the conversion tests elsewhere, let's add conversion tests for the other entity types here. Also would be good to add the wildcards in the middle of the entity name, in the parent entity name, etc. to make sure all possible cases work.

Missed adding wildcard in the middle of the entity name, in parent entity name, etc.